package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Alex's GraphQL project",
   "main": "index.js",
   "scripts": {
-    "start-dev": "nodemon src/index.js",
-    "start": "node src/index.js"
+    "start-dev": "node src/app.js && nodemon src/index.js",
```

Comment:
i think it's better to wrap the sequelize stuff in a function and call it from src/index.js so you are only running one file

Reply:
Combined both Sequelize and Apollo server initialization into index.js and modified the scripts.